src/ui/components/SecondaryToolbox/index.tsx

```diff
@@ -68,6 +73,15 @@ const PanelButtons: FC<PanelButtonsProps> = ({
           <div className="label">Elements</div>
         </button>
       )}
+      <button
+        className={classnames("console-panel-button", {
+          expanded: selectedPanel === "debugger",
+        })}
+        onClick={() => onClick("debugger")}
+      >
+        <div className="label">Sources</div>
```

Comment:
Considering that the actual "Sources" panel can still be visible, and that the item in the dropdown to hide it says "Hide Editor", we can name this "Editor". Seems more consistent.

Reply:
Yep the naming could certainly be improved. Sources is useful though because that's what chrome does